packages/color/src/index.js

```diff
@@ -1,7 +1,7 @@
 import * as P from 'polished'
 import { get } from '@styled-system/css'
 
-const g = (t, c) => get(t, `colors.${c}`, c)
+const g = (t, c) => get(t, `colors.${c}`, c).replace(/var\(--(\w+)(.*?), /,'').replace(/\)/,'')
```

Comment:
Awesome, thanks! Wondering if the regex should use `^`??
```suggestion
const g = (t, c) => get(t, `colors.${c}`, c).replace(/^var\(--(\w+)(.*?), /,'').replace(/\)/,'')
```

Reply:
Sure couldn't hurt.